ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+  - name: 'virtual_mailbox_base'
+    value: '{{ postldap__mailbox_base }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Optional lookup tables with all valid addresses in the domains that match $virtual_mailbox_domains.
+  - name: 'virtual_mailbox_maps'
+    value: 'ldap:/etc/postfix/ldap_virtual_recipients.cf'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Lookup tables with the per-recipient user ID that the virtual delivery agent uses while writing
+  # to the recipient's mailbox.
+  - name: 'virtual_uid_maps'
+    value: 'static:{{ postldap__vmail_posix_uidnumber }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Lookup tables with the per-recipient group ID for virtual mailbox delivery.
+  - name: 'virtual_gid_maps'
+    value: 'static:{{ postldap__vmail_posix_gidnumber }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # The minimum user ID value that the virtual delivery agent accepts as a result from $virtual_uid_maps table lookup
+  # FIXME: problems while parsing number.
+  - name: 'virtual_minimum_uid'
+    value: '{{ postldap__vmail_posix_uidnumber }}'
+    section: 'virtual'
+    state: 'absent'
+
+  # Optional lookup table with the SASL login names that own the sender (MAIL FROM) addresses.
+  - name: 'smtpd_sender_login_maps'
+    value: 'ldap:/etc/postfix/ldap_virtual_senders.cf'
+    section: 'base'
+    state: '{{ "present"
+            if (postldap__ldap_enabled|bool)
+            else "absent" }}'
+                                                                   # ]]]
+# .. envvar:: postldap__postfix_ldap_connection [[[
+#
+# The header of the lookup table containing the settings to connect to the LDAP server
+postldap__postfix_ldap_connection:
+  server_host: '{{ postldap__ldap_uri }}'
+  start_tls: '{{ "yes"
+                  if (postldap__ldap_start_tls|bool)
+                  else "no" }}'
+  version: '3'
+  tls_ca_cert_dir: '{{ postldap__tls_ca_cert_dir }}'
+  bind: 'yes'
+  bind_dn: '{{ postldap__ldap_binddn }}'
+  bind_pw: '{{ postldap__ldap_bindpw }}'
+  scope: 'sub'
+                                                                   # ]]]
+# .. envvar:: postldap__postfix__lookup_virtual_aliases [[[
+#
+# The virtual_alias_maps setting is used to find the target mailbox
+# (configured in the attribute mail, aka the mailbox or maildrop).
+postldap__postfix__lookup_virtual_aliases:
```

Comment:
The `postldap__postfix__lookup*` variables should be renamed to `postldap__postfix_lookup*` because they are not directly passed via the role dependent variables and are not directly used in the playbook.